### PR TITLE
Update DocumentPicker.getDocumentAsync API docs

### PR DESCRIPTION
**Pull request by BrunoBernardino**
# Why

It's not clear that the method returns a promise.

# How

I'm making it more clear that it returns a promise (even having "Async" in the name, this is clearly explained in other APIs like https://docs.expo.io/versions/v36.0.0/sdk/filesystem/#filesystemreadasstringasyncfileuri-options)

# Test Plan

N/A

### CODE REVIEW

File: docs/pages/versions/unversioned/sdk/document-picker.md [hunk 1]
```diff
@@ -55,6 +55,8 @@ Display the system UI for choosing a document. By default, the chosen file is co
 
 #### Returns
 
-If the user cancelled the document picking, returns `{ type: 'cancel' }`.
+Returns a promise that resolves to an object as explained below.
```
Comment (tsapeta) [inline docs/pages/versions/unversioned/sdk/document-picker.md]:
```suggestion
On success returns a promise that resolves to an object containing `{ type: 'success', uri, name, size }` where `uri` is a URI to the local document file, `name` is its original name and `size` is its size in bytes.
If the user cancelled the document picking, the promise resolves to `{ type: 'cancel' }`.
```

File: docs/pages/versions/unversioned/sdk/document-picker.md [hunk 2]
```diff
@@ -55,6 +55,8 @@ Display the system UI for choosing a document. By default, the chosen file is co
...
 
-Otherwise, returns `{ type: 'success', uri, name, size }` where `uri` is a URI to the local document file, `name` is its original name and `size` is its size in bytes.
+If the user cancelled the document picking, the promise returns `{ type: 'cancel' }`.
```
Comment (tsapeta) [inline docs/pages/versions/unversioned/sdk/document-picker.md]:
```suggestion
```

File: docs/pages/versions/unversioned/sdk/document-picker.md [hunk 3]
```diff
@@ -55,6 +55,8 @@ Display the system UI for choosing a document. By default, the chosen file is co
...
+
```
Comment (tsapeta) [inline docs/pages/versions/unversioned/sdk/document-picker.md]:
```suggestion
```

File: docs/pages/versions/unversioned/sdk/document-picker.md [hunk 4]
```diff
@@ -55,6 +55,8 @@ Display the system UI for choosing a document. By default, the chosen file is co
...
+Otherwise, it returns `{ type: 'success', uri, name, size }` where `uri` is a URI to the local document file, `name` is its original name and `size` is its size in bytes.
```
Comment (tsapeta) [inline docs/pages/versions/unversioned/sdk/document-picker.md]:
```suggestion
```

File: docs/pages/versions/v36.0.0/sdk/document-picker.md [hunk 5]
```diff
@@ -55,6 +55,8 @@ Display the system UI for choosing a document. By default, the chosen file is co
 
 #### Returns
 
-If the user cancelled the document picking, returns `{ type: 'cancel' }`.
+Returns a promise that resolves to an object as explained below.
```
Comment (tsapeta) [inline docs/pages/versions/v36.0.0/sdk/document-picker.md]:
```suggestion
On success returns a promise that resolves to an object containing `{ type: 'success', uri, name, size }` where `uri` is a URI to the local document file, `name` is its original name and `size` is its size in bytes.
If the user cancelled the document picking, the promise resolves to `{ type: 'cancel' }`.
```

File: docs/pages/versions/v36.0.0/sdk/document-picker.md [hunk 6]
```diff
@@ -55,6 +55,8 @@ Display the system UI for choosing a document. By default, the chosen file is co
...
 
-Otherwise, returns `{ type: 'success', uri, name, size }` where `uri` is a URI to the local document file, `name` is its original name and `size` is its size in bytes.
+If the user cancelled the document picking, the promise returns `{ type: 'cancel' }`.
```
Comment (tsapeta) [inline docs/pages/versions/v36.0.0/sdk/document-picker.md]:
```suggestion
```

File: docs/pages/versions/v36.0.0/sdk/document-picker.md [hunk 7]
```diff
@@ -55,6 +55,8 @@ Display the system UI for choosing a document. By default, the chosen file is co
...
+
```
Comment (tsapeta) [inline docs/pages/versions/v36.0.0/sdk/document-picker.md]:
```suggestion
```

File: docs/pages/versions/v36.0.0/sdk/document-picker.md [hunk 8]
```diff
@@ -55,6 +55,8 @@ Display the system UI for choosing a document. By default, the chosen file is co
...
+Otherwise, it returns `{ type: 'success', uri, name, size }` where `uri` is a URI to the local document file, `name` is its original name and `size` is its size in bytes.
```
Comment (tsapeta) [inline docs/pages/versions/v36.0.0/sdk/document-picker.md]:
```suggestion
```

**Review by tsapeta — APPROVED** [review]
Thanks for your contribution @BrunoBernardino! 🙏
Now sounds much better, but I'd reword it a bit more to make it even more clear 🙂
Suggestion below 👇

**Comment by BrunoBernardino** [comment]
Thanks @tsapeta that reads more clearly indeed!
